src/appmixer/utils/rss/NewFeedItem/component.json

```diff
@@ -24,16 +30,40 @@
         {
             "name": "out",
             "options": [
-                { "label": "Title", "value": "title" },
-                { "label": "Link", "value": "link" },
-                { "label": "Publication Date", "value": "pubDate" },
-                { "label": "Content", "value": "content" },
-                { "label": "Content Snippet", "value": "contentSnippet" },
-                { "label": "GUID", "value": "guid" },
-                { "label": "Categories", "value": "categories" },
-                { "label": "ISO Date", "value": "isoDate" }
+                {
+                    "label": "Title",
+                    "value": "title"
+                },
+                {
+                    "label": "Link",
+                    "value": "link"
+                },
+                {
+                    "label": "Publication Date",
+                    "value": "pubDate"
+                },
+                {
+                    "label": "Content",
+                    "value": "content"
+                },
+                {
+                    "label": "Content Snippet",
+                    "value": "contentSnippet"
+                },
+                {
+                    "label": "GUID",
+                    "value": "guid"
+                },
+                {
+                    "label": "Categories",
+                    "value": "categories"
+                },
+                {
+                    "label": "ISO Date",
+                    "value": "isoDate"
+                }
             ]
         }
     ],
     "icon": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjxzdmcgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiB3aWR0aD0iNTEyIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPjx0aXRsZS8+PHBhdGggZD0iTTEwOC41NiwzNDIuNzhhNjAuMzQsNjAuMzQsMCwxLDAsNjAuNTYsNjAuNDRBNjAuNjMsNjAuNjMsMCwwLDAsMTA4LjU2LDM0Mi43OFoiLz48cGF0aCBkPSJNNDgsMTg2LjY3djg2LjU1YzUyLDAsMTAxLjk0LDE1LjM5LDEzOC42Nyw1Mi4xMXM1Miw4Ni41Niw1MiwxMzguNjdoODYuNjZDMzI1LjMzLDMxMi40NCwxOTkuNjcsMTg2LjY3LDQ4LDE4Ni42N1oiLz48cGF0aCBkPSJNNDgsNDh2ODYuNTZjMTg1LjI1LDAsMzI5LjIyLDE0NC4wOCwzMjkuMjIsMzI5LjQ0SDQ2NEM0NjQsMjM0LjY2LDI3Ny42Nyw0OCw0OCw0OFoiLz48L3N2Zz4="
-}
+}
```

Comment:
It seems your IDE trims the last empty line.

Reply:
My apologies, I already made it so my IDE doesn't format on save, so this shouldn't happen in the future

Reply:
I have put the empty line back in the 1.0.1 commit